uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/include/uwrt_mars_rover_xbox_controller/coordinateNode.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef COORDINATENODE_HPP_
+#define COORDINATENODE_HPP_
+
+#include <uwrt_mars_rover_xbox_controller/visibility.h>
+
+#include <geometry_msgs/msg/twist.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <uwrt_mars_rover_xbox_controller/msg/xbox_controller.hpp>
+
+namespace drivetraincontrollerComposition
```

Comment:
Let's change the namespace to uwrt_comms
```suggestion
namespace uwrt_comms
```